crates/db_schema/src/source/local_user.rs

```diff
@@ -58,6 +58,8 @@ pub struct LocalUser {
   pub totp_2fa_enabled: bool,
   /// Whether to allow keyboard navigation (for browsing and interacting with posts and comments).
   pub enable_keyboard_navigation: bool,
+  /// Whether user avatars in the UI that are gifs should be allowed to play or should be paused
+  pub enable_animated_images: bool,
```

Comment:
Here too

Reply:
They both say user avatars and inline images.

Reply:
Yes its updated now